### ISSUE TITLE
Problem with installing prokka

### ISSUE THREAD

**Issue by NorthonColt** [tseemann/prokka#686]
I'm trying to install prokka on a linux server with the following specifications:
- Python version 3.11.7
- Perl version 5.38.2
- OS: Rocky Linux 8.9 (Green Obsidian)

For the installation I tried to use the specified command using conda:
`conda install -c conda-forge -c bioconda -c defaults prokka`
However, after running this code, and trying to check the version of the installed prokka (`prokka -version`) I get the error attached in _prokka_install_error.txt_.
[prokka_install_error.txt](https://github.com/tseemann/prokka/files/14411249/prokka_install_error.txt)

Thank you in advance for your help, it is very much appreciated.

Best regards,
Márton Réit

**Comment by m-abdullah-nabeel** [comment]
Hi!
As the error clearly shows `you may need to install the Bio::Root::Version module`, you are probably missing bioperl as one the dependencies. 
1. You may try to install bioperl manually as written in the docs of bioperl at https://github.com/bioperl/bioperl-live/blob/master/README.md#installation
2. Since you are already using conda, make sure you have activated the correct environment having all the dependies in it. You can also try installing bioperl via conda `conda install -c bioconda bioperl`.
3. Further make sure the all the installed dependencies are added to PATH. You can try `which` command for this purpose. Like `which python3` etc.
4. Finally, after checking all the dependencies are installed, try to reinstall according to the docs mentioned at https://github.com/tseemann/prokka#installation for your relevant environment. Or try to force re-install `conda install -c conda-forge -c bioconda -c defaults --force-reinstall prokka`.
In a nutshell, I think either you are missing the dependencies or they are not really added to the path.
